Apps/W1/ReviewGLEntries/app/src/tables/GLEntryReviewEntry.Table.al

```diff
@@ -16,6 +28,10 @@ table 22216 "G/L Entry Review Entry"
         {
             DataClassification = EndUserIdentifiableInformation;
         }
+        field(4; "Reviewed Amount"; Decimal)
```

Comment:
@Bertverbeek4PS you've added this field and therefore this field will be always zero on existing records.
I'm not sure if the data upgrade does really make sense this way.

Could you maybe help me understand why you added this field to the obsolete table?